frame/authorship/src/lib.rs

```diff
@@ -238,6 +238,7 @@ impl<T: Trait> Module<T> {
 
 			T::EventHandler::note_uncle(
 				author.clone().unwrap_or_default(),
+				// WARN `uncle` cannot be after `now`
```

Comment:
why is this true?

Reply:
ok - i guess these are *our* uncles.

Reply:
@gavofyork I was not sure.  That is why I included this message!

Reply:
I added an explicit check for this.